xml/operating_system_darwin.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Patterns for common names of Mac OS X operating systems.
+-->
+<fingerprints matches="operating_system.name">
+  <fingerprint pattern="^Mac OS X\s?(\d+?(?:\.\d+?)+?)?$" flags="REG_ICASE">
```

Comment:
So I think this will not match patterns like "Mac OS X 10", and "Mac OS 10.5". Should we make this a little less greedy. For example, how about making the "X" optional, and then using a * rather than + for the nested versions. I guess what I guess what I am favoring is a "weak" fingerprint vs no fingerprint.